docs/source/api/module.md

```diff
@@ -2,8 +2,146 @@ torchsynth.module
 =================
 
 ```{eval-rst}
-.. automodule:: torchsynth.module
-   :members:
-   :undoc-members:
-   :show-inheritance:
+SynthModule
```

Comment:
This is way more verbose I realize -- and we will need to update this every time we add a module. But this allows the theme to create a table of contents with modules grouped by audio rate, control rate, and parameter modules which I really like.

Reply:
There may be a way to template this out in the future. But I think this works for now.